download-libs.sh

```diff
@@ -42,6 +48,7 @@ if [ ! -d closure-compiler/.git ]; then
   fi
   git clone --depth 1 https://github.com/google/closure-compiler closure-compiler
```

Comment:
@daniel-ziegler is working on submodularizing this repo, so we can open a pull req upstream too